docs/release-notes/2.10.5.md

```diff
@@ -0,0 +1,3 @@
+## Bugfixes
+
+Reverted the previously released fix for the [issue](https://github.com/kyma-project/api-gateway/issues/1632), because it was adding an additional Authorization Policy with action ALLOW, which was hard to override.
```

Comment:
```suggestion
We have reverted the previously released fix for [issue #1632](https://github.com/kyma-project/api-gateway/issues/1632) because it added an additional AuthorizationPolicy with action `ALLOW`, which was difficult to override.
```